### ISSUE TITLE
Variables containing "_[0-9]" are not displayed

### ISSUE THREAD

**Issue by MatthiasJ1** [bernhard-42/vscode-ocp-cad-viewer#113]
```py
path_2d = ...
shape_3d = ...
_33 = ...
this_is_1_sphere = Sphere(1)

show_all()
```

Nothing will be displayed.

**Comment by bernhard-42** [comment]
I wanted to ensure that the `_i1`, `_i2`, ... of IPython/Jupyter are ignored. But chose the wrong regex.

I fixed it so that this happens:
```python
path_2d = Sphere(0.6)
shape_3d = Box(1,2,2)
_33 = Cylinder(0.3,4)
_i33 = Box(3,1,1)
this_is_1_sphere = Sphere(1)

show_all()
``` 
![image](https://github.com/user-attachments/assets/d58836c7-3351-412a-a2a2-745b2810343a)

Everything is shown, except `_i33` which is the ipython /Jupyter pattern `_i\d+`

**Comment by MatthiasJ1** [comment]
Awesome, thanks!

**Comment by bernhard-42** [comment]
Fixed in 2.6.1
